### PR TITLE
First contribution to HelloWorldJava

### PR DESCRIPTION
**Pull request by javithalion**
A couple of changes for the hello world file

### CODE REVIEW

File: src/javahelloworld/JavahelloWorld.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
      */
     public static void main(String[] args) {
         
-        System.out.println("Hello world");
+        System.out.println("Hello Fatima!"); // This prints Hello Fatima into the console.
```
Comment (FatimaRahmati) [inline src/javahelloworld/JavahelloWorld.java]: Comment is not necessary.

**Review by FatimaRahmati — CHANGES_REQUESTED** [review]

File: src/javahelloworld/JavahelloWorld.java (hunk 1 above)
Comment (javithalion) [reply src/javahelloworld/JavahelloWorld.java]: Are you sure?

Comment (FatimaRahmati) [reply src/javahelloworld/JavahelloWorld.java]: yes, I'm sure.

Comment (javithalion) [reply src/javahelloworld/JavahelloWorld.java]: Ok, I will remove this comment. Thanks for your feedback @FatimaRahmati

Comment (FatimaRahmati) [reply src/javahelloworld/JavahelloWorld.java]: Ok, thank you

**Pull request merged**